JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -144,29 +146,27 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
                                          missing = missing, callback = progressbarTick))
           
           relyFit$freq$est$freq_alpha <- Bayesrel:::applyalpha(model[["dat_cov"]])
-          p <- ncol(dataset)
-          Ctmp <- array(0, c(p, p - 1, p - 1))
-          for (i in 1:p){
-            Ctmp[i, , ] <- model[["dat_cov"]][-i, -i]
-          }
+          relyFit[["freq"]][["est"]] <- relyFit[["freq"]][["est"]][c(5, 1, 2, 3, 4)]
```

Comment:
Is it possible to somehow avoid the  statement `[c(5, 1, 2, 3, 4)]`? What is it doing? Can you perhaps reorder based on names on the elements? Asking just because this way is prone to errors if the output of the package changes (adds or deletes some elements)...

Reply:
Ja, I know this looks bad. The problem is that the order of output from the R-package is different from the order of the coefficients in the module. I would like to change this in the future, but for this PR, I won't. I hope that is okay.

Reply:
Sure, as a temporary quick fix it's okay.